arithmetization/src/main/java/net/consensys/linea/zktracer/module/oob/OobOperation.java

```diff
@@ -542,10 +542,11 @@ private BigInteger callToMOD(
 
   private boolean callToLT(
       int k, BigInteger arg1Hi, BigInteger arg1Lo, BigInteger arg2Hi, BigInteger arg2Lo) {
-    checkArgument(arg1Hi.bitLength() / 8 <= 16);
-    checkArgument(arg1Lo.bitLength() / 8 <= 16);
-    checkArgument(arg2Hi.bitLength() / 8 <= 16);
-    checkArgument(arg2Lo.bitLength() / 8 <= 16);
+    // 128 = 8 * 16
+    checkArgument(arg1Hi.bitLength() <= 128);
```

Comment:
I'd prefer to have a `LIMB_SIZE` constants in GlobalConstants.java, (so from the `constant/constants.lisp`)

Reply:
I mean, one LIMB_BYTE_SIZE` and one `LIMB_BIT_SIZE` to be explicit